errors/now-dev-as-dev-script.md

```diff
@@ -0,0 +1,9 @@
+# `now dev` as `dev` script
+
+#### Why This Error Occurred
+
+The `package.json` file of the used project invokes `now dev` as `dev` script. This can lead to `now dev` recursively calling itself.
```

Comment:
```suggestion
The `package.json` file of the used project invokes `now dev` as `dev` script. This would cause `now dev` to recursively invoke itself.
```